### ISSUE TITLE
Get-NotificationTrigger - There was an error deserializing the object of type PrtgAPI.NotificationTrigger

### ISSUE THREAD

**Issue by astapelfeld** [lordmilko/PrtgAPI#131]
The following error is generated when the sensor name contains quotation marks:

```powershell
PS > get-sensor -Id 5725 | Get-NotificationTrigger
Get-NotificationTrigger : There was an error deserializing the object of type PrtgAPI.NotificationTrigger. Encountered unexpected character 'E'.                                                                                          At line:1 char:23                                                                                                                                                                                                                         + get-sensor -Id 5725 | Get-NotificationTrigger                                                                                                                                                                                           +                       ~~~~~~~~~~~~~~~~~~~~~~~                                                                                                                                                                                           + CategoryInfo          : NotSpecified: (:) [Get-NotificationTrigger], SerializationException
+ FullyQualifiedErrorId : System.Runtime.Serialization.SerializationException,PrtgAPI.PowerShell.Cmdlets.GetNotificationTrigger
```

Sensor name: Disk Free: D:\ Label: Exchange "DATA"  -> Generates the error
Sensor name: Disk Free: D:\ Label: Exchange DATA -> No error

**Comment by lordmilko** [comment]
Hi @astapelfeld,

I was unable to reproduce this error by assigning the specified name to a sensor.

Can you please run the following commands and provide the output

```powershell
Set-PrtgClient -LogLevel Response

Get-Sensor -Id 5725 | Get-NotificationTrigger -Verbose

$error[0].exception.stacktrace
$error[0].exception.innerexception.stacktrace
```

This will emit the response that was actually returned from PRTG as well as show where exactly the failure occurred. I don't believe there should be any sensitive information in the emitted response, however you might want to check for any references to Notification Actions and modify the user's names to protect their privacy.

Please save the output from these commands in a text file and upload it to this issue

**Comment by astapelfeld** [comment]
[Get-NotificationTrigger_Error.txt](https://github.com/lordmilko/PrtgAPI/files/4249850/Get-NotificationTrigger_Error.txt)

**Comment by lordmilko** [comment]
Thanks @astapelfeld,

I have managed to reproduce the issue by creating a new state trigger on an object whose name contains a quote. At a glance it appears PRTG is (once again) transmitting completely illegal JSON/XML around

Specifically, the `href` attribute in the response above decodes to the following

```html
href=\"sensor.htm?id=5725\">Disk Free: D:&#92; Label: Exchange "DATA"</a>"
```

This is illegal; the quotes surrounding the word DATA should be double encoded so that they come out as single encoded when processed by a parser.

I will look into implementing a workaround for this. Thanks for raising this issue

**Comment by lordmilko** [comment]
Hi @astapelfeld,

I have released a pre-release build of PrtgAPI 0.9.13 that includes a fix for this issue. Can you please follow the [manual installation instructions](https://github.com/lordmilko/PrtgAPI/wiki/Installation#manual) and advise whether this build resolves the issue?

**Comment by astapelfeld** [comment]
Hi @lordmilko ,
The problem is solved in the pre-release. Thank you!
